### DISCUSSION TITLE
Custom chatbots

### DISCUSSION THREAD

**Discussion by Gunkev** [chatwoot/chatwoot#7248] [Q&A]
**Is your feature request related to a problem? Please describe.**
Custom chatbot support

**Describe the solution you'd like**
I want chatbots that can speak to users fluently without the need to create an intent for each chatbot.

**Describe alternatives you've considered**
I have used the Bing chatbot once and I noticed it could converse with a user fluently responding to questions in the context fluently without the need to create a bot intent. It would be nice if chatwoot could have this kind of out-of-the-box feature or if not Chatwoot could offer a custom solution for every developer/user so that they could save a lot of time

**Additional context**
Chatbots should not always be based on intent

**Comment by pranavrajs** [comment]
Moving this to discussions as the issue is too generic and there is no much to take actions.

[a comment by an automated account is not shown]
